src/ecs/rendering/systems.rs

```diff
@@ -54,14 +54,13 @@ impl<'a> System<'a> for RenderSystem {
                                 stage.set_target(new_target.clone());
                                 // Update stage effects, which are used to draw on the buffers
                                 for pass in stage.passes.iter_mut() {
-                                    let mut effect = &mut pass.effect.data;
+                                    let effect = &mut pass.effect.data;
                                     // Update Color Buffer
-                                    effect.out_colors.extend(
+                                    effect.out_colors = 
```

Comment:
`clear` + `extend` will reuse allocation. No big deal of course but why not.

Reply:
I search "rust vec remove all" and didn't find anything, thanks for saving me :D